### ISSUE TITLE
GAN train will hand if not use default GANTrainSteps(1, 1)

### ISSUE THREAD

**Issue by mpjlu** [tensorflow/tensorflow#22410]
When I test WGAN on tensorflow,  I set the generator_train_steps = 1 and discriminator_train_steps = 5 following the paper, like this:
tensorflow/contrib/gan/python/train.py:815
`def get_sequential_train_hooks(train_steps=namedtuples.GANTrainSteps(1, 5))`
The train process always hand after the first step. 
And the default value works well.
`def get_sequential_train_hooks(train_steps=namedtuples.GANTrainSteps(1, 1))`

Test:
just run 
`https://github.com/tensorflow/models/blob/master/research/gan/cifar/train.py `
with cifar10 dataset.

**Comment by mpjlu** [comment]
@joel-shor , do you have some comments about this? Thanks.

**Comment by harshini-gadige** [comment]
@joel-shor Hi, did you get a chance to look into this ?

**Comment by tensorflowbutler** [comment]
It has been 14 days with no activity and the `awaiting response` label was assigned. Is this still an issue?

**Comment by tensorflowbutler** [comment]
It has been 30 days with no activity and the `awaiting response` label was assigned. Is this still an issue?

**Comment by harshini-gadige** [comment]
> @joel-shor Hi, did you get a chance to look into this ?

Any update ?

**Comment by tensorflowbutler** [comment]
Nagging Assignee @joel-shor: It has been 14 days with no activity and this issue has an assignee. Please update the label and/or status accordingly.

**Comment by gowthamkpr** [comment]
@mpjlu Can you please take a look at the following implemntation of wgan [here](https://github.com/ChengBinJin/WGAN-GP-tensorflow/blob/master/README.md) and also please let me know if its still an issue. Thanks!
